profile.md

```diff
@@ -4,7 +4,7 @@
 David Perez-Suarez
 
 ## Backup administrator
-Stuart Mumford, Matthew, Erik, Tom
+Stuart Mumford, Matthew Turk, Tom Aldcroft, Kyle Barbary, Erik(?)
```

Comment:
Don't put me on it, please.  I *might* be able to, but how much time I have depends on whether or not I get any non-GSoC summer students at my current institution, so I can't commit to this yet...